ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/snapsync/SnapWorldDownloadStateTest.java

```diff
@@ -281,8 +281,6 @@ public void shouldRestartHealWhenNewPivotBlock() {
   public void shouldWaitingBlockchainWhenTooBehind() {
     when(snapSyncState.isHealInProgress()).thenReturn(true);
 
-    assertThat(snapSyncState.isWaitingBlockchain()).isFalse();
```

Comment:
Don't we need to assert this one anymore?

Reply:
it was trying to assert the return value of a different method on the mocked object - it doesn't actually make sense